packages/web3-eth-ens/src/abi/registry.ts

```diff
@@ -108,6 +116,7 @@ const REGISTRY = [
 		type: 'event',
 	},
 	{
+		constant: true,
```

Comment:
what does constant do?

Reply:
As per my knowledge `constant` is set to `true` if either function is `pure` or `view` and `false` otherwise. The fields `constant` and `payable` is deprecated in favor of `stateMutability`, but kept as backward compatibility.